hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -231,8 +231,11 @@ public ContainerCommandResponseProto sendCommand(
     try {
       return sendCommandWithTraceIDAndRetry(request, null).
           getResponse().get();
-    } catch (ExecutionException | InterruptedException e) {
+    } catch (ExecutionException e) {
       throw new IOException("Failed to execute command " + request, e);
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
```

Comment:
Can we log an error here? Same apply to other places?

Reply:
Would you prefer something like:
`LOG.error("Execution was interrupted", e)` ?

Reply:
@xiaoyuyao addressed review comment in recent commit.

Reply:
Same as the below comment. Can you verify once?

Reply:
This is still missing?